src/Caliburn.Micro.Platform/ActionMessage.cs

```diff
@@ -374,20 +374,46 @@ void ElementUnloaded(object sender, RoutedEventArgs e)
         /// <returns>The matching method, if available.</returns>
         public static Func<ActionMessage, object, MethodInfo> GetTargetMethod = (message, target) => {
 #if WINDOWS_UWP
-            return (from method in target.GetType().GetRuntimeMethods()
-                    where method.Name == message.MethodName
-                    let methodParameters = method.GetParameters()
-                    where message.Parameters.Count == methodParameters.Length
-                    select method).FirstOrDefault();
+            //return (from method in target.GetType().GetRuntimeMethods()
```

Comment:
could we remove the commented out code?  If we need to see old code it should be in the history for the repo